html/changelogs/shedya - dressshirts.yml

```diff
@@ -0,0 +1,40 @@
+################################
+# Example Changelog File
+#
+# Note: This file, and files beginning with ".", and files that don't end in ".yml" will not be read. If you change this file, you will look really dumb.
+#
+# Your changelog will be merged with a master changelog. (New stuff added only, and only on the date entry for the day it was merged.)
+# When it is, any changes listed below will disappear.
+#
+# Valid Prefixes: 
+#   bugfix
+#   wip (For works in progress)
+#   tweak
+#   soundadd
+#   sounddel
+#   rscadd (general adding of nice things)
+#   rscdel (general deleting of nice things)
+#   imageadd
+#   imagedel
+#   maptweak
+#   spellcheck (typo fixes)
+#   experiment
+#   balance
+#################################
+
+# Your name.  
+author: VTCobaltblood
+
+# Optional: Remove this file after generating master changelog.  Useful for PR changelogs that won't get used again.
+delete-after: True
+
+# Any changes you've made.  See valid prefix list above.
+# INDENT WITH TWO SPACES.  NOT TABS.  SPACES.
+# SCREW THIS UP AND IT WON'T WORK.
+# Also, all entries are changed into a single [] after a master changelog generation. Just remove the brackets when you add new entries.
+# Please surround your changes in  double quotes ("), as certain characters otherwise screws up compiling. The quotes will not show up in the changelog.
+changes: 
+  - rscadd: "Added dress shirts to loadout. Are an accessory, recolorable, significantly resprited from chef's uniform. Also added dress shirts with rolled up sleeves."
+  - rscadd: "Added a casual skirt and a long skirt to loadout. Both are recolorable."
+  - rscadd: "Added recolorable suit jackets to loadout, resprited from IAA suit jackets."
+  - tweak: "Removed items made obsolete by these additions (khaki skirt, blue/purple suit jacket) from loadout. They still can be spawned in."
```

Comment:
This entire changelog should be shortened but it is no big deal.

Reply:
Into one rscadd? I can do that.